python/ml4ir/base/model/architectures/fixed_additive_positional_bias.py

```diff
@@ -0,0 +1,37 @@
+import tensorflow as tf
+from tensorflow.keras import layers
+
+
+class FixedAdditivePositionalBias(layers.Layer):
+    """
+    Implementing positional bias handling by learning a fixed additive weights
+    """
+    def __init__(self, max_ranks):
+        super(FixedAdditivePositionalBias, self).__init__()
+        self.dense = layers.Dense(1,
+                     name="fixed_additive_positional_bias_layer",
```

Comment:
should we use the name you defined in the `config`? `FIXED_ADDITIVE_POSITIONAL_BIAS_LAYER`?